modules/registry-configuring-registry-storage-swift-trust.adoc

```diff
@@ -7,13 +7,21 @@
 [id="registry-configuring-registry-storage-swift-trust_{context}"]
 = Configuring the Image Registry Operator to trust Swift storage
 
-You must configure the Image Registry Operator to trust {rh-openstack-first} Swift storage.
+You can configure the Image Registry Operator to trust {rh-openstack-first} Swift storage. This is an optional procedure depending on how your clients trust storage certificates.
 
+[NOTE]
+====
+In situations where clients to not trust the storage certificate, the `disableRedirect` option can be set to `true` to proxy traffic through the image registry. Consequently, however, the image registry requires significantly more resources, especially network bandwidth, to handle the increased load. 
```

Comment:
Nit:
```suggestion
In situations where clients to not trust the storage certificate, the `disableRedirect` option can be set to `true` to proxy traffic through the image registry. Consequently, however, the image registry may require significantly more resources, especially network bandwidth, to handle the increased load. 
```

If the image registry is not heavily used and the images in storage are not huge then disabling redirect might not cause any performance impact. So "may" seems more accurate.